src/gtk/toga_gtk/widgets/button.py

```diff
@@ -17,11 +17,18 @@ def set_label(self, label):
         self.rehint()
 
     def set_enabled(self, value):
-        # self._impl.set_sensitive(value)
-        self.interface.factory.not_implemented('Button.set_enabled()')
+        self.native.set_sensitive(value)
```

Comment:
Do we need to raw value or the processed value here? Usually, I'd expect to see `self.interface.enabled`

Reply:
[set_sensitive()](https://developer.gnome.org/gtk3/stable/GtkWidget.html#gtk-widget-set-sensitive) takes a boolean value. Both `value` and `self.interface.enabled` work just fine. (Tried both). Do you then mean `self.interface.enabled` for better readability?